saltstack/base/salt/ccmv2/cdp/bin/ccmv2/generate-config.sh

```diff
@@ -16,13 +16,12 @@ BACKEND_PORT=$3
 AGENT_KEY_PATH=$4
 AGENT_CERT_PATH=$5
 ENVIRONMENT_CRN=$6
-ACCESS_KEY_ID=$7
-ACCESS_KEY=$8
-TRUSTED_BACKEND_CERT_PATH=$9
-TRUSTED_PROXY_CERT_PATH=${10}
-INVERTING_PROXY_URL=${11}
+ACC_KEY_TMP_FILE=$7
```

Comment:
Nit: I suggest following the naming convention used for `AGENT_KEY_PATH` etc., so renaming this to `ACC_KEY_PATH` (or `ACCESS_KEY_PATH`).